docs/core/tools/sdk-errors/index.md

```diff
@@ -339,4 +352,18 @@ This is a complete list of the errors that you might get from the .NET SDK while
 |NETSDK1190|To use '{0}' in solution projects, you must set the environment variable '{1}' (to true). This will increase the time to complete the operation.|
 |NETSDK1191|A runtime identifier for the property '{0}' couldn't be inferred. Specify a rid explicitly.|
 |NETSDK1192|Targeting .NET 7.0 or higher in Visual Studio 2022 17.3 is not supported.|
+|NETSDK1193|If PublishSelfContained is set, it must be either true or false. The value given was '{0}'.|
+|NETSDK1194|The "--output" option isn't supported when building a solution. Specifying a solution-level output path results in all projects copying outputs to the same directory, which can lead to inconsistent builds.|
 |[NETSDK1195](netsdk1195.md)|Trimming, or code compatibility analysis for trimming, single-file deployment, or ahead-of-time compilation is not supported for the target framework. For more information, see <https://aka.ms/netsdk1195>|
+|NETSDK1196|The SDK does not support ahead-of-time compilation. Set the PublishAot property to false.|
+|NETSDK1197|Multiple solution project(s) contain conflicting '{0}' values; ensure the values match. Consider using a Directory.build.props file to set the property for all projects. Conflicting projects: {1}.|
+|NETSDK1198|A publish profile with the name '{0}' was not found in the project. Set the PublishProfile property to a valid file name.|
+|NETSDK1199|The ArtifactsPath and UseArtifactsOutput properties cannot be set in a project file, due to MSBuild ordering constraints. They must be set in a Directory.Build.props file or from the command line. See https://aka.ms/netsdk1199 for more information.|
```

Comment:
@dsplaisted Do you want to add more information for NETSDK1199?

Reply:
Not at this time, I think.